### PR TITLE
Feature/delete btn

### PR DESCRIPTION
**Pull request by brendel-r**
I added functionality to the Delete button in JS.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -51,15 +55,21 @@ function addIdeaCard(idea) {
...
     ideas.push(idea);
     addIdeaCard(idea);
     clearInputFields();
     toggleButton();
+    console.log("does save btn work?")
 }
+  function deleteIdea(event) {
+    if(event.target.className === "delete-btn") {
+      event.target.parentNode.parentNode.remove()
+      console.log("does this delete btn work?")
```
Comment (Denzel-Turambi) [inline main.js:70]: add `;`'s

**Review by RandyGitProjects — COMMENTED** [review]
Need to remove some console logs and update the styling for the delete button image.

File: main.js [hunk 2]
```diff
@@ -51,15 +55,21 @@ function addIdeaCard(idea) {
...
+    }
+  }
 
 function clearInputFields(){
```
Comment (asutherland91) [inline main.js:74]: For consistancy there should be a space between the () and {

**Review by asutherland91 — APPROVED** [review]
Aside from a few small edits it looks fantastic

**Pull request merged**